pywhat/Data/regex.json

```diff
@@ -727,6 +727,19 @@
          "Credentials",
          "Password",
          "Username"
-       ]
-    }
-]
+      ]
+   },
+   {
+      "Name": "PGP Public Key",
+      "Regex": "/^(-----BEGIN PGP PUBLIC KEY BLOCK-----).*([a-zA-Z0-9//\n\/\\.\\:\\+\\ \\=]+).*(-----END PGP PUBLIC KEY BLOCK-----)$|^(-----BEGIN PGP PRIVATE KEY BLOCK-----).*([a-zA-Z0-9//\n\/\\.\\:\\+\\ \\=]+).*(-----END PGP PRIVATE KEY BLOCK-----)$/",

```

Comment:
Could you move private key block into its own? I think it would be better if there would be two types - "PGP Public Key" for public keys and "PGP Private Key" for private keys.

Reply:
Thanks, just fixed it